README.md

```diff
@@ -103,6 +103,19 @@ Hutch::Logging.logger = Rails.logger
 See this [RabbitMQ tutorial on topic exchanges](http://www.rabbitmq.com/tutorials/tutorial-five-ruby.html)
 to learn more.
 
+### Message Processing Tracers
+
+Tracers allow you to track message processing. You can set them using `tracer` method.
+
+#### NewRelic
+```ruby
+class FailedPaymentConsumer
+  include Hutch::Consumer
+  consume 'gc.ps.payment.failed'
+  tracer 'NewRelic'
```

Comment:
Can we turn the argument here to be a class? I see zero value in it being a string.